lib/adapters/autocomplete-adapter.js

```diff
@@ -96,6 +96,7 @@ export default class AutocompleteAdapter {
       text: item.insertText || item.label,
       displayText: item.label,
       filterText: item.filterText || item.label,
+      snippet: item.insertTextFormat == 2 ? item.insertText : undefined,
```

Comment:
Sorry, my mistake. It's supposed to be `===` to keep the linter happy.  (JS not the language I'm most familiar with either ;-)  )